sources/Session.php

```diff
@@ -57,7 +57,10 @@ function loadSession()
 		// This is here to stop people from using bad junky PHPSESSIDs.
 		if (isset($_REQUEST[session_name()]) && preg_match('~^[A-Za-z0-9,-]{16,64}$~', $_REQUEST[session_name()]) == 0 && !isset($_COOKIE[session_name()]))
 		{
-			$session_id = md5(md5('elk_sess_' . time()) . mt_rand());
+			require_once(SUBSDIR . '/TokenHash.class..php');
```

Comment:
Double dot! :P 
On a general note: isn't the autoloader able to deal with the class without having to rely on the requires?